src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -614,13 +615,11 @@ public static String swapCase(final String str) {
         for (int index = 0; index < strLen;) {
             final int oldCodepoint = str.codePointAt(index);
             final int newCodePoint;
-            if (Character.isUpperCase(oldCodepoint)) {
+            if (Character.isUpperCase(oldCodepoint) || Character.isTitleCase(oldCodepoint) ) {
```

Comment:
Nice catch. Any chance you could add a unit test for that too?

Reply:
The corresponding lines as the whole method are already completely covered by existing Unit Tests.
![selection_050](https://user-images.githubusercontent.com/6312834/30752358-5a0d0094-9fbc-11e7-9ada-822cb344cefb.png)